### ISSUE TITLE
Introduce sentry

### ISSUE THREAD

**Issue by warjiang** [karmada-io/dashboard#122]
**What would you like to be added**:
Introduce sentry for solving user web-ui problem.

**Why is this needed**:
Sentry is very useful for collecting errors、 reproduce problem especially for web.

**Comment by RainbowMango** [comment]
This [sentry](https://sentry.io/welcome/)?

**Comment by warjiang** [comment]
Yes, here is the sentry for open-source: https://sentry.io/for/open-source/

**Comment by warjiang** [comment]
The user reported the error stacktrace is hard to identify the problem, sentry will help us to parse the error stacktrace according to corresponded sourcemap

**Comment by warjiang** [comment]
Maybe we need extra config for whether to enable sentry, if users concered about the security problem, they can just disable the switch for sentry

**Comment by yashpandey06** [comment]
@RainbowMango is this issue resolved?

**Comment by RainbowMango** [comment]
Not yet, as far as I know.

I think the #147 is working on it.
